src/components/ui/BaseCard.vue

```diff
@@ -128,4 +135,9 @@ function formatDate(date: Date): string {
 function toggleDropDown(dropDownClicked: boolean) {
   isDropDownClicked.value = dropDownClicked;
 }
+
+function onTodoChecked() {
```

Comment:
why do you need a separate emit for todo checked? cant you use update-todo here?